### DISCUSSION TITLE
look bro thats someone take your bot copy paste it make it in top.gg and they name is Farah-Bot report it and git it out

### DISCUSSION THREAD

**Discussion by anxlz** [jagrosh/MusicBot#615] [Questions] [answered]
look bro thats someone take your bot copy paste it make it in top.gg and they name is Farah-Bot report it and git it out

**Comment by jagrosh** [comment, marked as answer]
If you see someone misusing the code in this repository or breaking the license, please report it to the site that you see the misuse on (in this case, you would need to report it to top.gg). Keep in mind that commercial use of the code _is_ allowed, assuming that the license (Apache 2.0) is followed.
